src/ray/gcs/gcs_server/gcs_node_manager.cc

```diff
@@ -24,26 +37,28 @@ void GcsNodeManager::HandleHeartbeat(const ClientID &node_id,
 
 void GcsNodeManager::Start() {
   RAY_LOG(INFO) << "Starting gcs node manager.";
-  const auto lookup_callback = [this](Status status,
-                                      const std::vector<GcsNodeInfo> &node_info_list) {
-    for (const auto &node_info : node_info_list) {
-      if (node_info.state() != rpc::GcsNodeInfo::DEAD) {
-        // If there're any existing alive clients in client table, add them to
-        // our `heartbeats_` cache. Thus, if they died before monitor starts,
-        // we can also detect their death.
-        // Use `emplace` instead of `operator []` because we just want to add this
-        // client to `heartbeats_` only if it has not yet received heartbeat event.
-        // Besides, it is not necessary to add an empty `HeartbeatTableData`
-        // to `heartbeat_buffer_` as it doesn't make sense to broadcast an empty
-        // message to the cluster and it's ok to add it when actually receive
-        // its heartbeat event.
-        heartbeats_.emplace(ClientID::FromBinary(node_info.node_id()),
-                            num_heartbeats_timeout_);
-      }
-    }
-    Tick();
-  };
-  RAY_CHECK_OK(gcs_client_->Nodes().AsyncGetAll(lookup_callback));
+  const auto lookup_callback =
```

Comment:
Hmm I'm not clear on what these changes are for. Was there a bug before?

Reply:
> Hmm I'm not clear on what these changes are for. Was there a bug before?

I just add this line `AddNode(std::make_shared<rpc::GcsNodeInfo>(std::move(node_info)));` and I guess that this changes is caused by clang-format